### PR TITLE
155 add navigation buttons to about and how to play

### PR DESCRIPTION
**Pull request by kajgrant**
- Added buttons to the bottom of the home and about pages

### CODE REVIEW

File: client/src/views/About.vue [hunk 1]
```diff
@@ -21,5 +21,8 @@
         <a v-t="'about.coffee'" href="https://www.buymeacoffee.com/ConorMurphy/" />
       </template>
     </i18n-t>
+    <router-link to="/">
+      <button v-t="'homeButton'" type="button" class="btn btn-orange w-10" />
```
Comment (ConorMurphy21) [inline client/src/views/About.vue:25]: all buttons should have a sound played when pressed, you can look at other buttons for examples on how to do this

**Review by ConorMurphy21 — CHANGES_REQUESTED** [review]

**Review by jengu288 — CHANGES_REQUESTED** [review]

File: client/src/views/About.vue (hunk 1 above)
Comment (kajgrant) [reply client/src/views/About.vue]: I used the default click1.mp3, let me know if there is a different sound that should be used.

File: client/src/views/About.vue [hunk 2]
```diff
@@ -21,5 +21,23 @@
         <a v-t="'about.coffee'" href="https://www.buymeacoffee.com/ConorMurphy/" />
       </template>
     </i18n-t>
+    <router-link to="/" @click="onClick()">
+      <button v-t="'homeButton'" type="button" class="btn btn-burgundy mt-3 w-25" />
+    </router-link>
   </div>
 </template>
+
+<script lang="js">
```
Comment (ConorMurphy21) [inline client/src/views/About.vue:30]: preferably set lang to ts

**Review by ConorMurphy21 — APPROVED** [review]
everything looks good except about can be typescript. How to play has to stay js because i18n has a bug in their typing

File: client/src/views/About.vue (hunk 2 above)
Comment (kajgrant) [reply client/src/views/About.vue]: done

**Review by jengu288 — APPROVED** [review]

**Pull request merged**